libcontainerd/remote_linux.go

```diff
@@ -254,6 +256,11 @@ func (r *remote) handleEventStream(events containerd.API_EventsClient) {
 	for {
 		e, err := events.Recv()
 		if err != nil {
+			if strings.Contains(err.Error(), "transport is closing") &&
```

Comment:
I was hoping to find a way to inspect this error without resorting to checking the error string, but it doesn't seem that this is possible with this error, which is a `grpc.rpcError`

Maybe we can do `strings.Contains(err.Error(), transport.ErrConnClosing.Desc)`